cloud/src/meta-service/meta_service_job.cpp

```diff
@@ -1509,18 +1510,29 @@ void MetaServiceImpl::finish_tablet_job(::google::protobuf::RpcController* contr
     recorded_job.ParseFromString(job_val);
     VLOG_DEBUG << "get tablet job, tablet_id=" << tablet_id
                << " job=" << proto_to_json(recorded_job);
+    FinishTabletJobRequest_Action action = request->action();
 
-    std::unique_ptr<int, std::function<void(int*)>> defer_commit(
-            (int*)0x01, [&ss, &txn, &code, &msg, &need_commit](int*) {
-                if (!need_commit) return;
-                TxnErrorCode err = txn->commit();
-                if (err != TxnErrorCode::TXN_OK) {
-                    code = cast_as<ErrCategory::COMMIT>(err);
-                    ss << "failed to commit job kv, err=" << err;
-                    msg = ss.str();
-                    return;
+    std::unique_ptr<int, std::function<void(int*)>> defer_commit((int*)0x01, [&ss, &txn, &code,
+                                                                              &msg, &need_commit,
+                                                                              &action](int*) {
+        if (!need_commit) return;
+        TxnErrorCode err = txn->commit();
+        if (err != TxnErrorCode::TXN_OK) {
+            if (err == TxnErrorCode::TXN_CONFLICT) {
+                if (action == FinishTabletJobRequest::COMMIT) {
+                    g_bvar_delete_bitmap_lock_txn_remove_conflict_by_compaction_commit_counter << 1;
```

Comment:
is it needed for none mow table?

Reply:
Even though none mow doesn't have delete bitmap lock conflict, it may have some other confict, i think it's better to record too.